### ISSUE TITLE
Replicability of your Research

### ISSUE THREAD

**Issue by GiancarloPaoletti** [panji1990/Deep-subspace-clustering-networks#12]
Greetings, first of all let me say that I read your paper with interest. 
Thanks for having shared the code! I'd like to address you some question about it.

1. By running the code as you provided it (without changes), I am able to almost perfectly reproduce your published results (e.g., 14% error on ORL as in Figure 5.a). This happens **if** I select the max value of accuracy among 100 random runs, out of which I get a mean accuracy of 84.5%, with a standard deviation of 1.2% on the same benchmark. I was using the **same** hyper-parameters of yours. Can we then conclude that you followed a similar procedure to obtain the results inserted in the paper? 
2. In the _README.md_, you correctly mentioned about the usage of the diagonal constraint on the self-expressive module C (diag(C)=0) - the latter being fundamental if using L1 regularization. To implement such constraint, I used the snippet of code that you provide therein (`tf.matmul((C-tf.diag(tf.diag_part(C))),Z)`). I put such line of code - once I changed the variable names - replacing line 48 inside _DSC-Net-L2-ORL.py_. Then, I replaced line 59 of the same file with 
`self.reg_losses = tf.reduce_sum(tf.math.abs(self.Coef))`.
Unfortunately, if I do so, the results with L1 drastically differs from the ones tabulated in the paper. I'd like to ask you if you have an updated version of the code (maybe already implementing the diag constraint + L1), so that I can check and compare.
3. Also, I'm encountering an out-of-memory error when running the code on the COIL100 dataset. Can I kindly ask which are the specs of the computer/server you used for experiments (in terms of RAM mainly)? 

Thank you very much for your attention
Looking forward to your reply
